website/src/components/layout/sidebar/styles.less

```diff
@@ -5,25 +5,23 @@
   background: #f4f4f4;
   overflow-y: scroll;
   width: 300px;
-  padding: 3rem 2rem;
+  padding: 3rem 2rem 0 2rem;
 
   .typo-systemtittel {
     margin-bottom: 1rem;
   }
 
   .contribute-promo {
     padding: 2rem 2rem 2rem 2rem;
-    margin: 2rem -2rem -3rem -2rem;
+    margin: 2rem -2rem 0 -2rem;
     border-top: 1px solid @navGra20;
   }
 
   .divider {
     margin-top: 1rem;
   }
 
-  &--components {
-    li:last-child {
-      margin-top: 1rem;
-    }
+  &__input {
+    margin-bottom: -1rem;
```

Comment:
Jeg ser på negative margins som en code smell. Noen få ganger er det nødvendig, men det er unntaket. Uten å ha sett så nøye på dette, så regner jeg med det her kan fikses med en ryddigere layout-styling, sannsynligvis med `display: grid`. Men vi kan godt si at å rydde i det er utenfor scope for denne PRen.

Reply:
Enig i at det er dårlig praksis, endret løsning.